CRITICAL - 2020-10-27 03:45:20 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

#0 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\anime\web2\app\Models\Video_Model.php(211): CodeIgniter\Database\BaseConnection->query('SELECT\r\n       ...')
#3 C:\xampp\htdocs\anime\web2\app\Controllers\Home.php(72): App\Models\Video_Model->get_anime('1')
#4 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(910): App\Controllers\Home->player('1', '0')
#5 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\anime\web2\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-10-27 03:45:20 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\anime\web2\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\anime\web2\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Home), true)
#2 C:\xampp\htdocs\anime\web2\system\Debug\Exceptions.php(298): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\anime\web2\system\Debug\Exceptions.php(171): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-10-27 03:46:17 --> A connection attempt failed because the connected party did not properly respond after a period of time, or established connection failed because connected host has failed to respond.

#0 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(377): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(643): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\anime\web2\app\Models\Video_Model.php(211): CodeIgniter\Database\BaseConnection->query('SELECT\r\n       ...')
#3 C:\xampp\htdocs\anime\web2\app\Controllers\Home.php(72): App\Models\Video_Model->get_anime('1')
#4 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(910): App\Controllers\Home->player('1', '0')
#5 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#6 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\anime\web2\index.php(45): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-10-27 03:46:17 --> Uncaught ErrorException: print_r(): Property access is not allowed yet in C:\xampp\htdocs\anime\web2\app\Views\errors\html\error_exception.php:100
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler(2, 'print_r(): Prop...', 'C:\\xampp\\htdocs...', 100, Array)
#1 C:\xampp\htdocs\anime\web2\app\Views\errors\html\error_exception.php(100): print_r(Object(App\Controllers\Home), true)
#2 C:\xampp\htdocs\anime\web2\system\Debug\Exceptions.php(298): include('C:\\xampp\\htdocs...')
#3 C:\xampp\htdocs\anime\web2\system\Debug\Exceptions.php(171): CodeIgniter\Debug\Exceptions->render(Object(mysqli_sql_exception), 500)
#4 [internal function]: CodeIgniter\Debug\Exceptions->exceptionHandler(Object(mysqli_sql_exception))
#5 {main}
  thrown
#0 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
#1 {main}
CRITICAL - 2020-10-27 03:46:42 --> Table 'anime8k.mo_movie' doesn't exist
#0 C:\xampp\htdocs\anime\web2\system\Database\MySQLi\Connection.php(330): mysqli->query('SELECT\r\n       ...')
#1 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT\r\n       ...')
#2 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT\r\n       ...')
#3 C:\xampp\htdocs\anime\web2\app\Models\Video_Model.php(211): CodeIgniter\Database\BaseConnection->query('SELECT\r\n       ...')
#4 C:\xampp\htdocs\anime\web2\app\Controllers\Home.php(72): App\Models\Video_Model->get_anime('1')
#5 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(910): App\Controllers\Home->player('1', '0')
#6 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\anime\web2\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-10-27 03:46:58 --> Table 'anime8k.mo_movie' doesn't exist
#0 C:\xampp\htdocs\anime\web2\system\Database\MySQLi\Connection.php(330): mysqli->query('SELECT\r\n       ...')
#1 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT\r\n       ...')
#2 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT\r\n       ...')
#3 C:\xampp\htdocs\anime\web2\app\Models\Video_Model.php(211): CodeIgniter\Database\BaseConnection->query('SELECT\r\n       ...')
#4 C:\xampp\htdocs\anime\web2\app\Controllers\Home.php(72): App\Models\Video_Model->get_anime('1')
#5 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(910): App\Controllers\Home->player('1', '0')
#6 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\anime\web2\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2020-10-27 03:47:34 --> Table 'anime8k.mo_movie' doesn't exist
#0 C:\xampp\htdocs\anime\web2\system\Database\MySQLi\Connection.php(330): mysqli->query('SELECT\r\n       ...')
#1 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(738): CodeIgniter\Database\MySQLi\Connection->execute('SELECT\r\n       ...')
#2 C:\xampp\htdocs\anime\web2\system\Database\BaseConnection.php(666): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT\r\n       ...')
#3 C:\xampp\htdocs\anime\web2\app\Models\Video_Model.php(211): CodeIgniter\Database\BaseConnection->query('SELECT\r\n       ...')
#4 C:\xampp\htdocs\anime\web2\app\Controllers\Home.php(72): App\Models\Video_Model->get_anime('1')
#5 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(910): App\Controllers\Home->player('1', '0')
#6 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(398): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Home))
#7 C:\xampp\htdocs\anime\web2\system\CodeIgniter.php(306): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\xampp\htdocs\anime\web2\index.php(45): CodeIgniter\CodeIgniter->run()
#9 {main}
